tests/io/dask/bag/test_read.py

```diff
@@ -65,3 +65,31 @@ def test_read_dataset_as_dataframes_partition_size(store_factory, metadata_versi
             dataset_uuid="partitioned_uuid", store=store_factory, partition_size=2
         )
         assert bag.npartitions == 2
+
+
+def test_read_dataset_as_dataframe_bag_multi_table_warning(store_factory, metadata_version):
```

Comment:
This test doesn't verify the warning, does it?

Reply:
I am on it to verify it with https://docs.pytest.org/en/stable/warnings.html#warns